enhancements/installer/aws-lb-subnet-selection.md

```diff
@@ -0,0 +1,1147 @@
+
+To achieve this, this enhancement deprecates the existing install-config field `platform.aws.subnets`
+in favor of a more flexible configuration field that handles specifying subnets for IngressControllers,
+control plane load balancers, and the cluster nodes. These new subnet roles, referred to as
+IngressControllerLB, ControlPlaneExternalLB, ControlPlaneInternalLB, ClusterNode, and EdgeNode subnets
+are further defined in [Defining Subnet Roles](#defining-subnets-roles).
+
+## Definitions and Terminology
+
+### The `default` IngressController
+
+This proposal will refer to the IngressController that gets created automatically during installation and handles
+the platform routes (console, auth, canary, etc.) as the `default` IngressController. Note that
+"default IngressController subnets" refers to the default subnets used by all IngressControllers, including
+the `default` IngressController. This is different from "`default`-IngressController subnets", which specifically
+means the subnets for the `default` IngressController.
+
+### User-Created IngressControllers
+
+A user-created IngressController is a custom IngressController that is created after installation (Day 2).
+It represents all IngressControllers except for the `default` IngressController.
+
+### Defining Subnets Roles
+
+With the install-config now able to specify multiple ways a subnet could be used, it's important
+to clarify what these uses are. We will define the function, purpose, or use of a subnet as a "role".
+We use the concept of roles to recognize that a single subnet can simultaneously fulfill multiple roles.
+
+Let's define the new subnet roles:
+
+| Role                   | Description                                                                                                                                                                                                                                                                                                                      | Allowed Subnet Scope                                        | 
+|------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| ClusterNode            | Host the cluster nodes, including both control plane and compute nodes in standard availability zones (excluding local zones).                                                                                                                                                                                                   | Private [^1]                                                |
```

Comment:
Nit:

```suggestion
| ClusterNode            | Subnet hosts the cluster nodes, including both control plane and compute nodes in standard availability zones (excluding local zones).                                                                                                                                                                                                   | Private [^1]                                                |
```
Similar Nit for other `Description`. Only because `Host` could mean a compute resource too.